parquet/src/main/java/org/apache/iceberg/parquet/ParquetBloomRowGroupFilter.java

```diff
@@ -48,8 +48,12 @@
 import org.apache.parquet.schema.LogicalTypeAnnotation.DecimalLogicalTypeAnnotation;
 import org.apache.parquet.schema.MessageType;
 import org.apache.parquet.schema.PrimitiveType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ParquetBloomRowGroupFilter {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ParquetBloomRowGroupFilter.class);
```

Comment:
Minor: Can we add an empty line after this line to separate the static and instance variables?

Reply:
Added. Thanks